### DISCUSSION TITLE
Could not purchase this item: can't have a retired listing plan

### DISCUSSION THREAD

**Discussion by jameswilddev** [renovatebot/renovate#8014] [Suggest an Idea]
Hello!

On trying to add Renovate to my public org, I see a red banner containing:

"Could not purchase this item: can't have a retired listing plan"

Have I done something wrong?

Thanks,
  James.

**Comment by rarkins** [comment]
The Renovate App was converted to be a free-only app, so anyone on paid plans was migrated. I don't know exactly what's going on (what you're seeing is a GitHub error, not Renovate) but you could try going to https://github.com/marketplace/renovate and removing any plan there and then re-adding. If the problem persists, contact GitHub to ask them to resolve. Either way Renovate should be running on all installed repos irregardless of whether any marketplace plan is selected.

**Comment by jameswilddev** [comment]
Ah, figured it out.

![image](https://user-images.githubusercontent.com/3138458/102224394-d0666c80-3edd-11eb-9e88-fd961f7721f5.png)

In this section, if nothing down the left is highlighted in blue, click on one - this indicates that the plan you're on is no longer available and that you need to pick on the from the left.

GitHub UI problem.
